packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -1176,6 +1195,7 @@ operations will be generated by the CLI.`,
     parent: ObjectTypeDefinitionNode | null,
     modelConfiguration: ModelDirectiveConfiguration,
     explicitOperationName: string = undefined,
+    updateSchema: boolean = true,
```

Comment:
nit: should include same description for `updateSchema` argument used in `protectGetQuery`

Reply:
Updated the doc string